backend/src/app.ts

```diff
@@ -39,15 +63,29 @@ class App {
   }
 
   private routes(): void {
-    const { modelsArray } = getAllModelsInDir(path.join(__dirname, "./routes"));
-    modelsArray.forEach((route: any) => {
-      this.express.use("/api", route);
+    routes.forEach((route) => {
+      console.log(route);
+      this.express.use(`/api/${route.group}`, route.router);
```

Comment:
`route.group` assumes, a more strict rule of the array-based route definition, can't we work with an easier approach or use the previous setup.
I think just remove the loop, and just add this ` this.express.use("/api", routes);`